requirements.txt

```diff
@@ -1,4 +1,4 @@
-ladybug-geometry==1.23.0
+ladybug-geometry==1.23.1
```

Comment:
As you can see here this approach will make dependency conflict: https://github.com/ladybug-tools/honeybee-vtk/pull/29/checks?check_run_id=1773999859#step:4:40

Since honeybee-core already has dependencies on ladybug-core (https://github.com/ladybug-tools/honeybee-core/blob/master/requirements.txt) and ladybug-core has dependencies on ladybug-geometry (https://github.com/ladybug-tools/ladybug/blob/master/requirements.txt) you can remove ladybug-geometry and only keep updating the honeybee-core dependency.